issue #767 user guide update for job auto-restart (#1183)
IBM/FHIR: docs/src/pages/guides/FHIRBulkOperations.md

### **$bulkdata-status: Poll Job**
The response returned is 202 if the job is queued or not yet complete.

> (Note: if the job is stopped, e.g, due to shutdown of the fhir server instance which runs the bulkdata JavaBatch job, then upon receiving the status polling request, the bulkdata JavaBatch job will be restarted and continue the import/export with 202 response to the client.)

Reviewer: ```suggestion

Note: If the job is stopped, e.g, due to shutdown of the IBM FHIR Server where the job is running the bulkdata job, then upon polling the status of the bulkdata job, the server restarts the JavaBatch job, and the response to the client is Accepted: 202.)
```
